packages/visualizations-react/package.json

```
@@ -52,7 +52,7 @@
         "arrowParens": "avoid"
     },
     "dependencies": {
-        "@opendatasoft/visualizations": "0.24.1-beta.2",
+        "@opendatasoft/visualizations": "^0.24.1",
```

```suggestion
        "@opendatasoft/visualizations": "0.24.1",
```

If I remember,  we had issue without the caret